vi/dot_file.py

```diff
@@ -31,20 +31,20 @@ def run(self):
     def parse(self, line):
         try:
             _logger().info('[DotFile] parsing line: {0}'.format(line))
-            if line.startswith((':map ')):
-                line = line[1:]
+            line = line.lstrip(':')

```

Comment:
Question: does Vim allow spaces before `:` in config files? If so, we'd need to account for it here.